lib/cucumber.rb

```diff
@@ -9,6 +9,7 @@
 module Cucumber
   class << self
     attr_accessor :wants_to_quit
+    attr_accessor :treats_ambiguous_as_fullwidth
```

Comment:
I also suggest we rename it to `treat_ambiguous_width_characters_as_double_width` to be crystal clear.